string object_id
int64 position_x
int64 position_y
int64 size_x
int64 size_y
